src/backer/cardaning.py

```diff
@@ -104,122 +98,130 @@ def flushQueue(self):
+            funding_payment_verification_key = pycardano.PaymentVerificationKey.from_signing_key(funding_payment_signing_key)
+            funding_addr = pycardano.Address(funding_payment_verification_key.hash(), None, network=NETWORK)
         except KeyError:
             print("Backer address could not be funded. Environment variable FUNDING_ADDRESS_CBORHEX is not set")
             return
+        
         try:
-            fund_addr = funding_addr.encode()
-            resp = self.api.address(address=fund_addr)
-            funding_balance = resp.amount[0]
-        except ApiError as e:
-            if e.status_code == 404:
-                print("Empty balance to fund backer")
-            else:
-                print("Blockfrost API error:",e.message)
+            fund_addr = funding_addr.encode()            
+            funding_balance = self.getaddressBalance(fund_addr)
+        except Exception as e:
+            logger.info("Empty balance to fund backer")
             return
             
         print("Funding address:", funding_addr)
-        print("Funding balance:", int(funding_balance.quantity)/1000000,"ADA")
-        if int(funding_balance.quantity) > (FUNDING_AMOUNT + 1000000):
-            try:
-                builder = TransactionBuilder(self.context)
+        print("Funding balance:", int(funding_balance)/1000000,"ADA")
+
+        if int(funding_balance) > (FUNDING_AMOUNT + 1000000):
+            try:                                
+                builder = pycardano.TransactionBuilder(self.context)
                 builder.add_input_address(funding_addr)
-                builder.add_output(TransactionOutput(addr,Value.from_primitive([int(FUNDING_AMOUNT/3)])))
-                # builder.add_output(TransactionOutput(addr,Value.from_primitive([int(FUNDING_AMOUNT/3)])))
-                # builder.add_output(TransactionOutput(addr,Value.from_primitive([int(FUNDING_AMOUNT/3)])))
-                signed_tx = builder.build_and_sign([funding_payment_signing_key], change_address=funding_addr)
-                self.context.submit_tx(signed_tx.to_cbor())
+                builder.add_output(pycardano.TransactionOutput(addr, pycardano.Value.from_primitive([int(FUNDING_AMOUNT/3)])))
+                signed_tx = builder.build_and_sign([funding_payment_signing_key], change_address=funding_addr)                
+                self.context.submit_tx(signed_tx)
                 print("Funds submitted. Wait...")
-                time.sleep(50)
-                balance = self.getaddressBalance()
+                time.sleep(50)                
+                balance = self.getaddressBalance(self.spending_addr.encode())
+                
                 if balance:
                     print("Backer balance:",balance/1000000, "ADA")
             except Exception as e:
-                print("error", e)
+                logger.debug(traceback.format_exc())
+                print("Error: Cannot fund backer", e)
```

Comment:
we have a mix of logger and print. Can we just use logger? Here we could use `logger.critical` instead as its a big issue

Reply:
I removed the "print" and use ogger.critical instead